app/views/media/_external_link_card.html.erb

```diff
@@ -5,9 +5,10 @@
     </h5>
   </div>
   <div class="card-body text-center">
-    <%= link_to t('click_here'),
+    <%= link_to external_link_description_not_empty(medium),
                 medium.external_reference_link,
```

Comment:
I am not sure if it is visually the most pleasing to have a very large button that visually dominates the whole page if the description text is quite long, but I also do not have an alternative idea. Mabye just plain the text and then a simple "Link"-Button?
![button](https://user-images.githubusercontent.com/28628554/234899196-11979a23-4253-4d2c-876d-fb21141b59dc.png)

Reply:
I thought the display text was just intended for some short text, e.g. "LA Bosch". That's why the input field is also just a small field and not a whole paragraph. The `word-break: break-all;` is therefore just for the edge case, so that the button does not stick out of the outer div. (Note that edge case also means when the display text is empty, then we show the url itself).

While I don't think that this button is overly pretty for long texts, I also don't really like the option of a "Link" button next to the display text as I feel like links should show the thing they point to directly on them and not somewhere else.